doc-Infrastructure_Migration_Solution_Guide/cfme/common/attributes/cfme.adoc

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
I just noticed this file only has IMS attributes, so should the file name instead be ims.adoc instead of cfme.adoc? 

It also looks like it's still going to an in-guide-only common directory, rather than manageiq_docs/common/attributes/ (which works ok if you only ever need it to appear in this guide, but if it ever needs to be used elsewhere in CloudForms, or if we ever start sharing content for installation instead of using links, then it won't be sufficient). 

Alternatively, if you don't see it being used outside this guide, then it doesn't really need to be in any common directory, and can just live in the cfme/ directory with master.adoc.

Reply:
I hadn't thought of that, because I copied an existing cfme doc to create the v2v guide. I've changed the file name and link.